addons/schedule/model-java/src/main/java/org/commonjava/indy/schedule/datastax/ScheduleDBUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.commonjava.indy.schedule.datastax;
+
+public class ScheduleDBUtil
+{
+
+    public static final String TABLE_SCHEDULE = "schedule";
+
+    public static String getSchemaCreateTableSchedule( String keyspace )
+    {
+        return "CREATE TABLE IF NOT EXISTS " + keyspace + "." + TABLE_SCHEDULE + " ("
+                        + "jobtype varchar,"
+                        + "jobname varchar,"
+                        + "storekey varchar,"
+                        + "scheduletime timestamp,"
+                        + "lifescan bigint,"
+                        + "expiration varchar,"
+                        + "PRIMARY KEY (storekey, jobname)"
+                        + ");";
+    }
+
+    public static String getSchemaCreateTypeIndex4Schedule( String keyspace )
+    {
+        return "CREATE INDEX IF NOT EXISTS type_idx on " + keyspace + "." + TABLE_SCHEDULE + " (jobtype)";
+    }
+
```

Comment:
Adding the storekey as partition key, and the jobtype as one of the secondary index, then we can do the following two queries:
`select * from schedule where jobtype='Disable-timeout'`
`select * from schedule where storekey='' and jobtype='CONTENT'`

Seems there are just two types "Disable-timeout" and "CONTENT", so setting the jobtype as the index maybe not a good way, but let's try it first.